addons/destruction/destruction.gd

```diff
@@ -101,12 +101,12 @@ func _add_shard(original: MeshInstance3D, explosion_power: float) -> void:
 			-original.position.normalized())
 	if shrink_delay < 0 and fade_delay < 0:
 		get_tree().create_timer(animation_length)\
-				.timeout.connect(func(): body.queue_free())
+				.timeout.connect(func(): if is_instance_valid(body): body.queue_free())
```

Comment:
I think it would make the most sense to use a tween in both of these branches, and use `Tween.tween_method` here.